playbooks/openstack.yml

```diff
@@ -66,7 +66,7 @@
 - name: Configure operating system
   hosts: controllers:computes
   become: true
-  gather_facts: false
+  gather_facts: true
```

Comment:
This is the default, so let's just remove the line :)

Reply:
thanks